andela/the_spinners-frontend
# #169817485 Reset password via email

### What does this PR do?
- Has to find user
- Has reset password
#### Description of Task to be completed?
- When a user forgets its password he will get an option to recover
#### How should this be manually tested?
- Click on forget password
- Enter your email
- In your email inbox, click reset password link
- When redirected to reset password page 
- Insert new password and confirm to reset password
#### What are the relevant pivotal tracker stories?
[#169817485](https://www.pivotaltracker.com/story/show/169817485)
#### Screenshots
<img width="640" alt="Screen Shot 2020-01-30 at 09 39 14" src="https://user-images.githubusercontent.com/38500196/73429459-6c2dcf00-4344-11ea-8ac6-f945acf4faac.png">
